kubb.config.ts

```diff
@@ -0,0 +1,71 @@
+import { defineConfig } from '@kubb/core'
+import { pluginClient } from '@kubb/plugin-client'
+import { pluginOas } from '@kubb/plugin-oas'
+import { pluginTs } from '@kubb/plugin-ts'
+
+const config = defineConfig({
+  root: '.',
+  input: {
+    path: 'https://api.indexcoop.com/v2/docs/json',
+  },
+  output: {
+    path: './src/gen',
+    clean: true,
+  },
+  plugins: [
+    pluginOas({
+      generators: [],
+    }),
+    pluginTs({
+      output: {
+        path: 'models',
+        barrelType: 'propagate',
+        banner: `/* eslint-disable */`,
+      },
+      exclude: [
+        {
+          type: 'tag',
+          pattern: 'admin',
+        },
+      ],
+      group: {
+        type: 'tag',
+      },
+      enumType: 'asConst',
+      dateType: 'date',
+    }),
+    pluginClient({
+      baseURL: 'https://api-q513.onrender.com',
+      output: {
+        path: './clients/axios',
+        barrelType: 'propagate',
+        banner: `/* eslint-disable */`,
+      },
+      exclude: [
+        {
+          type: 'tag',
+          pattern: 'admin',
+        },
+      ],
+      importPath: '@/lib/axios',
+      group: {
+        type: 'tag',
+        name({ group }) {
+          return `${group}Service`
+        },
+      },
+      operations: true,
+      pathParamsType: 'object',
+      dataReturnType: 'full',
+    }),
+    // pluginTanstackQuery({
```

Comment:
In case we want to access api endpoints which are public from the frontend this can be useful, but of course, we can just write it with a useQuery as well.